src/app/modules/main/module.nim

```diff
@@ -1649,4 +1596,23 @@ method checkIfAddressWasCopied*[T](self: Module[T], value: string) =
     return
   self.addressWasShown(value)
 
+method createMemberItem*[T](self: Module[T], memberId: string, state: MembershipRequestState, role: MemberRole): MemberItem =
```

Comment:
Did you forget to create the forward declaration?

Reply:
Maybe it's because of the `io_interface`?

Reply:
maybe... I don't have deep knowledge of NIM, but during dev testing, everything is ok + this should be a compilation error

Reply:
Indeed. All's good then